lib/gollum/app.rb

```diff
@@ -25,6 +24,10 @@
 require 'gollum/views/rss.rb'
 require 'gollum/views/template_cascade'
 
+['sassc', 'sassc-embedded'].each do |gem|
+  require gem if Gem::Specification.find {|spec| spec.name == gem}
```

Comment:
If not running via `bundle exec`, these gems may not be available unless installed manually, so only require them conditionally.